src/tests/Microsoft.PowerFx.Interpreter.Tests/DisplayNameOptionSetTests.cs

```diff
@@ -47,6 +48,42 @@ public void OptionSetDisplayNames(string inputExpression, string outputExpressio
             }
         }
 
+        // Verify methods to go between OptionSet/Type/Value
+        [Fact]
+        public void TestHelpers()
```

Comment:
>TestHelpers [](http://example.com/codeflow?start=20&length=11)

We could move this to Core if we moved OptionSet....
(or the test could add its own IExternalOptionSet impl)